src/libraries/filcrypto/filproofs/algorithms.go

```diff
@@ -204,7 +204,7 @@ func (f *Feistel_I) Permute(size UInt, i UInt) UInt {
 	panic("TODO")
 }
 
-func (sdr *WinStackedDRG_I) Seal(sid sector.SectorID, data []byte, randomness sector.SealRandomness) SealSetupArtifacts {
+func (sdr *WinStackedDRG_I) Seal(sid sector.SectorID, data []byte, randomness util.Randomness) SealSetupArtifacts {
```

Comment:
Why did you remove these types everywhere? This seems like a pure loss of information and makes the filproofs code less clear.

Reply:
More than less clear: it will make it harder to use the spec to communicate (and ensure) correctness.

Reply:
fixed. sorry